raiden/tests/unit/test_pfs_integration.py

```diff
@@ -296,11 +306,18 @@ def test_routing_mocked_pfs_bad_http_code(chain_state, token_network_state, our_
         address3: NODE_NETWORK_REACHABLE,
     }
 
-    # channel 1 and 2 are flipped here, to see when the PFS gets called
+    # in case the pfs sends a bad hhtp code but the correct path back
```

Comment:
Is getting the correct path but a wrong code something that is tested here?

Reply:
in that test, we mock the response code to 400 and the answer as the correct path. We see in the asserts that internal routing is used due to the bad code, right?

Reply:
Makes sense!

Reply:
Should we remove that test?

Reply:
No, it makes sense.